manage

```diff
@@ -599,27 +602,64 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
   #exit the manage script with the same exit-code of the app's script
   exit $exitcode
   
-elif [ "$1" == 'update' ];then #The manage script no longer handles updates. This mode is only for backwards-compatibility and uses the updater script
-  
-  warning "The manage script ONLY updates apps, and this mode has been replaced by the updater script.
-If you want to update Pi-Apps from the command-line, please use:
-~/pi-apps/updater cli-yes"
-  
+elif [ "$1" == 'update' ];then
```

Comment:
It seems unnecessary to duplicate the app update logic. Do you intend to remove the `update_app()` function from `updater`?

Reply:
the `update_app` function is still used by the CLI updater, so for now it has to stay.

I might change the `update_app` function so that it can be used within the manage update. I had made some very minor changes to the implementation in the manage script that made it so I couldn't use the version from the updater script

Reply:
resolved with https://github.com/Botspot/pi-apps/pull/2340/commits/1bb40287c7822ea1136da2e6648d10edda4db0fb